Index.html

```diff
@@ -214,9 +214,58 @@ <h2 class="hh2">Multi-Post Story</h2>
           </div>
 
       </article>
-      </main>
-      <footer>
+      <article class="contact-section" id="contact-section">
+        <section class="cc-sect cc-top">
+            <p class="contact mble">CONTACT US</p>
+            <p class="text-invite">
+                If you have an application you are interested in developing, 
+                a feature that you need built or a project that needs coding. 
+                I’d love to help with it 
+            </p>
+        </section>
 
+        <section class="cc-sect">
+            <form action="https://formspree.io/f/mgeqvqbb" method="post" class="myform" id="someform">
+                <input type="text" maxlength="30" name="Name" required placeholder="Full Name">
+                <input type="email" name="Email" required placeholder="Email" id="email">
+                <textarea maxlength="500" name="message" required placeholder="Enter You Message Here"></textarea>
+                <button type="submit" class="mmb">See Project</button>
+            </form>
```

Comment:
- [ ] Your form section is appearing twice on your website because the code is duplicated. kindly remove one of them 👍🏽